BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -218,9 +198,16 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
         guard let self = self else { return }
         publisherView.checkAgainButton.isLoading = true
         
-        self.state.ledger.refreshPublisher(withId: host, completion: { _ in
+        self.state.ledger.refreshPublisher(withId: host, completion: { [weak self] didRefresh in
+          guard let self = self else { return }
+          
           publisherView.checkAgainButton.isLoading = false
-          publisherView.checkAgainButton.isHidden = true
+          
+          if didRefresh {
```

Comment:
The bool that comes back from this is not `didRefresh` its `isVerified`, so not sure if this logic is correct?

Reply:
Fixed.